.goreleaser.yml

```diff
@@ -1,49 +1,42 @@
 # Visit https://goreleaser.com for documentation on how to customize this behavior.
+
 before:
   hooks:
-    # this is just an example and not a requirement
-    # - go mod tidy
+    # - 'go mod tidy'
+
```

Comment:
keep comment?

Reply:
yes, let's keep the comment (or delete it completely) because we have committed `go.mod` and `go.sum` which will be used (we pin to module versions which we test manually).
We don't want to download the latest versions of all the modules on each build b/c it could break something.